schemaorg.json

```diff
@@ -0,0 +1,100 @@
+{
+  "@context": "http://schema.org",
+  "@graph": [
+    {
+      "type": "SoftwareSourceCode",
+      "author": [
+        {
+          "id": "https://orcid.org/0000-0002-2815-0399"
+        },
+        {
+          "id": "https://orcid.org/0000-0002-1642-628X"
```

Comment:
People should only be listed with ID alone if they are also listed somewhere else in the document with a more complete entry (think of it as a reference).  So the maintainer could almost always be listed by id alone, since they will be an author as well and listed there, but authors probably need to be listed out.  Does that make sense?